content/scalability/docs/cluster-services.md

```diff
@@ -0,0 +1,60 @@
+# Cluster Services
+
+Cluster services run inside an EKS cluster, but they are not user workloads. If you have a Linux server you often need to run services like NTP, syslog, and a container runtime to support your workloads. Cluster services are similar, supporting services that help you automate and operate your cluster. In Kubernetes these are usually run in the kube-system namespace and some are run as DaemonSets.
+
+Cluster services are expected to have a high up-time and are often critical during outages and for troubleshooting. They should run on dedicated compute instances such as a separate node group or AWS Fargate. This will ensure that the cluster services are not impacted on shared instances by workloads that may be scaling up or using more resources.
+
+## Scale CoreDNS
+
+Scaling CoreDNS has two primary mechanisms. Reducing the number of calls to the CoreDNS service and increasing the number of replicas.
+
+### Reduce external queries by lowering ndots
+
+The ndots setting specifies how many periods in a domain name are considered enough to avoid querying DNS. If your application has an ndots setting of 5 (default) and you request resources from api.example.com (http://api.example.com/) (2 dots) then CoreDNS will be queried for each search domain defined in /etc/resolv.conf for a more specific domain (e.g. api.example.com.namespace.svc.) until all searches are exhausted or a match is found.
+
+You can reduce the number of requests to CoreDNS by lowering the ndots option (https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config) of your workload or fully qualifying your domain requests by including a trailing . (e.g. api.example.com.). If your workload connects to external services via DNS we recommend setting ndots to 2 so workloads do not make unnecessary, cluster DNS queries inside the cluster. You can set a different DNS server and search domain if the workload doesn’t require access to services inside the cluster.
```

Comment:
When you say to use a trailing dot, do you mean in your application code? If so, a code snippet would help.

Reply:
The highlighting in that doc refers to syntax highlighting for specific code languages. I tried using the `==highlight==` syntax in this doc https://squidfunk.github.io/mkdocs-material/reference/formatting/#highlighting-changes but it didn't work. I'm guessing we either don't have it configured or aren't using the right version of mkdocs